src/components/transactions/TransactionRow.vue

```diff
@@ -311,24 +307,16 @@ export default {
         data.txDataHash = await retry3Times(
           bridgeContract.methods.transactionsDataHashes(data.transaction.transactionHash).call,
         )
-        if (data.txDataHash != NULL_HASH) {
+        if (data.txDataHash !== NULL_HASH) {
           data.estimatedTime = ''
           const claimed = await retry3Times(bridgeContract.methods.claimed(data.txDataHash).call)
           if (claimed) {
             data.currentStep = data.steps.Claimed
-            if (data.isSenderAddress) {
-              TXN_Storage.addOrUpdateTxn(
-                data.sharedState.accountAddress,
-                data.fromNetwork.localStorageName,
-                { currentStep: data.currentStep, ...data.transaction },
-              )
-            } else if (data.isReceiverAddress) {
-              TXN_Storage.addOrUpdateTxn(
-                data.sharedState.accountAddress,
-                data.toNetwork.localStorageName,
-                { currentStep: data.currentStep, ...data.transaction },
-              )
-            }
+            await this.$services.TransactionService.saveTransaction({
```

Comment:
why do we have no distinction between the storage service for receiver and sender addresses?
just curious, since before we used two different local storage instances